### DISCUSSION TITLE
Theme launched with FL - Inconsistency at launch, using the hotkey "Alt+Sace" vs "SystemTray Icon"

### DISCUSSION THREAD

**Discussion by PCO62** [Flow-Launcher/Flow.Launcher#2611] [Q&A] [answered]
**Hello**

Maybe it's a very basic question, related to definitions, but for which I can find neither explanation nor solution (my ignorance!?)

**_Context Info:_**
===========
i) OS Version: Windows 11 Pro, 23H2
ii) FL Version: 1.17.2
iii) I have my personalized themes in the appropriate folder (_C:\...\...\AppData\Roaming\FlowLauncher\Themes_) confirmed by Settings>Appearance>Theme Folder> Open Theme Folder.
iv) **The selected theme** (Settings>Appearance>Theme) _**is one of the custom themes.**_
v) The general FL launch hotkey is the default: "Alt+Space"
vi) FL is predefined to start at startup and the respective icon appears in System Tray

**_Problem:_**
========
i) If I launch FL via "**_Alt+Space_**", the theme that is launched is **one of the program's predefined themes.**
ii) If I launch FL via the "**_SystemTray icon_**", the theme that is launched is **the selected custom theme.**

**_Question_**:
========
**_What could be the reason for this launching inconsistency of themes?_**

_I appreciate your feedback._
**Thanks**

**Comment by taooceros** [comment, marked as answer]
Got screenshot?

**Reply by PCO62** [reply]
I already managed to solve the problem (that's why I can't present any screenshots!), but without knowing what caused it! I hate these situations where we can't know the causes of a problem (because it could happen again)! :\

How did I solve it?... I simply reinstalled FL!

**_But thank you, @taooceros, for trying to help me!_**

**Reply by taooceros** [reply]
Alright because it doesn't make sense to me. Maybe you also install power toys?

**Reply by PCO62** [reply]
> Maybe you also install power toys?

Yes, I've PowerToys installed!

**Reply by PCO62** [reply]
Do you think that PowerToys can cause problems to FL?

**Reply by taooceros** [reply]
It's possible that you launch power toy run instead of flow.

**Reply by PCO62** [reply]
> It's possible that you launch power toy run instead of flow.

Are you referring to the PowerToys "sub-app" "**_PowerToys Run_**"? 
In fact, the associated shortcut was defined as "Alt+Space" (now changed!)!

After all, perhaps the "**_cause of the problem_**" (which I mentioned above) has been solved!

**Thank you @taooceros! ; )**

**Reply by taooceros** [reply]
You are welcome!
